src/model/Ticket.java

```diff
@@ -1,87 +1,110 @@
 package model;
 
 import enums.SectorType;
+import util.IdGenerator;
 
 import java.math.BigDecimal;
 import java.time.LocalDateTime;
 import java.time.format.DateTimeFormatter;
 import java.util.Currency;
 import java.util.Locale;
+import java.util.Objects;
 
-public class Ticket {
-    private String id;
-    private String concertHall;
-    private String eventCode;
+public class Ticket implements Printable,Identifiable {
```

Comment:
can't find setter for time field. please check, perhaps it was accidentally removed